### PR TITLE
[GSOC][TMVA][SOFIE] Swish Activation function implemented in the Keras Parser

### PR DESCRIPTION
**Pull request by Neel-Shah-29**
# This Pull request: Adds a Swish Activation function to the Keras Parser as required, given that swish operator does not exist in ONNX documentation.

## Checklist:

- [X] tested changes locally
- [ ] updated the docs (if necessary)

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build!

**Review by sanjibansg — APPROVED** [review]
LGTM!
Thank you for your contribution.

File: tmva/pymva/test/TestRModelParserKeras.C [hunk 1]
```diff
@@ -411,6 +412,45 @@ TEST(RModelParser_Keras, ACTIVATIONS)
     }
 }
 
+TEST(RModelParser_Keras, SWISH)
+{
+    constexpr float TOLERANCE = DEFAULT_TOLERANCE;
+    float input[]={1,1,1,1,1,1,1,1};
+    TMVA_SOFIE_swish_model::Session s("KerasSwish_model.dat");
+    std::vector<float> output = s.infer(input);
+
```
Comment (sanjibansg) [inline tmva/pymva/test/TestRModelParserKeras.C:421]: Many of the Keras tests, especially on the activation functions work in the same way, we can maybe try to generalize them to reduce code repetitions.

Comment (lmoneta) [reply tmva/pymva/test/TestRModelParserKeras.C:421]: Yes, we should do this, but it can be a separate PR

**Pull request merged**